### ISSUE TITLE
fine-tuning resnet-18 on UCF

### ISSUE THREAD

**Issue by ananth-kv** [kenshohara/3D-ResNets-PyTorch#47]
Hi, great work! Thank you.

Could you please tell me how long it took for you to fine-tune resnet-18(pretrained on kinetics) on UCF101 to get the reported accuracy (~84%)? Also, was there any specific hyperparameter setting I need to change while fine-tuning except freezing conv layers?

-Ananth

**Comment by kenshohara** [comment]
I evaluated the model trained at 300 epoch, and it got 84% accuracy.
This is the opts.json when I trained the model.
```
{
    "root_path": "../../../UCF101/",
    "video_path": "../../../UCF101/videos/jpg",
    "annotation_path": "../../../UCF101/ucf101_01.json",
    "result_path": "../../../UCF101/results/ft/resnet18_1e-03_1e-05_4",
    "dataset": "ucf101",
    "n_classes": 400,
    "n_finetune_classes": 101,
    "sample_size": 112,
    "sample_duration": 16,
    "sample_stride": 16,
    "initial_scale": 1,
    "n_scales": 5,
    "scale_step": 0.84089641525,
    "learning_rate": 0.001,
    "momentum": 0.9,
    "weight_decay": 0.00001,
    "nesterov": false,
    "optimizer": "sgd",
    "batch_size": 128,
    "n_epochs": 300,
    "begin_epoch": 1,
    "n_val_samples": 3,
    "resume_path": "",
    "pretrain_path": "../../../UCF101/models/resnet-18-kinetics.pth",
    "ft_begin_index": 4,
    "no_train": false,
    "no_val": false,
    "test": true,
    "test_subset": "val",
    "test_merge": "avg",
    "test_scale": 1,
    "no_cuda": false,
    "n_threads": 12,
    "checkpoint": 10,
    "crop": "c",
    "no_hflip": false,
    "norm_value": 1,
    "model": "resnet",
    "model_depth": 18,
    "resnet_shortcut": "A",
    "wide_resnet_k": 2,
    "resnext_cardinality": 32,
    "manual_seed": 1,
    "scales": [
        1,
        0.84089641525,
        0.7071067811803005,
        0.5946035574934808,
        0.4999999999911653
    ],
    "arch": "resnet-18",
    "mean": [
        114.7748,
        107.7354,
        99.475
    ]
}
```

**Comment by ananth-kv** [comment]
Awesome. Thanks so much.

Is it fine to assume that you evaluated at 300 epochs for all types of resnets while fine-tuning?

And 84% accuracy is **video-level** accuracy and not clip-level. right?

**Comment by kenshohara** [comment]
I think so.
I reported the results of the models at 300 epochs in my paper,
and the training and validation accuracies saturated at 200 epochs or earlier.

Right, the reported accuracy is video-level.

**Comment by slighting666** [comment]
@kenshohara @kvananth 
Hi, I got the accuracy of val.log, but I want to calculate the accuracy of video level, where can I get the val.json file and how to use eval_ucf101.py?
